src/components/events/EventCard/style.module.scss

```diff
@@ -0,0 +1,52 @@
+.container {
+  background: transparent;
+  border: 1px solid var(--theme-primary-6);
+  border-radius: 10px;
+  -ms-overflow-style: none;
+  overflow-x: hidden;
+  padding: 0;
+  position: relative;
+  scrollbar-width: none;
```

Comment:
Is there a reason why a scrollbar would appear?

Reply:
removed 👍